### PR TITLE
fix: [CDS-53660]: modified to use api calls for nexus groupid and artifactid

### PR DESCRIPTION
**Pull request by sahithikolichala**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/Nexus2Artifact/Nexus2Artifact.tsx [hunk 1]
```diff
@@ -492,11 +630,52 @@ export function Nexus2Artifact({
...
+                          }
+
+                          refetchArtifacts({
+                            queryParams: {
+                              ...commonParams,
+                              connectorRef: getConnectorRefQueryData(),
+                              repository: formik.values?.repository,
+                              repositoryFormat: formik.values?.repositoryFormat,
+                              groupId: formik.values?.spec?.groupId,
+                              nexusSourceType: 'Nexus3Registry'
```
Comment (pankajsingh-harness) [inline src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/Nexus2Artifact/Nexus2Artifact.tsx:674]: Make this `Nexus2Registry` and the API will start working fine.

**Comment by sahithikolichala** [comment]
retrigger jest

**Comment by sahithikolichala** [comment]
retrigger jest

**Review by pankajsingh-harness — CHANGES_REQUESTED** [review]

**Comment by pankajsingh-harness** [comment]
Do not merge this PR until delegates are updated. https://harness.atlassian.net/browse/CDS-58560

**Comment by sahithikolichala** [comment]
Closing because of merge conflicts

**Pull request closed without merging**